phpcs.xml.dist

```diff
@@ -202,9 +209,16 @@
 
     <rule ref="PSR2.Methods.MethodDeclaration.Underscore">
         <!-- extending a class from another package -->
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/DatabasePlatformMock.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/SchemaManagerMock.php</exclude-pattern>
         <exclude-pattern>tests/Doctrine/Tests/ORM/Functional/Ticket/DDC3634Test.php</exclude-pattern>
     </rule>
 
+    <rule ref="Squiz.NamingConventions.ValidVariableName.PublicHasUnderscore">
+        <!-- the impact of changing this would be too big -->
+        <exclude-pattern>tests/Doctrine/Tests/DbalFunctionalTestCase.php</exclude-pattern>
```

Comment:
What's the public underscored name?

Reply:
`$_conn`. Although it is protected, that rule triggers.

Reply:
See here why: https://github.com/squizlabs/PHP_CodeSniffer/blob/0aeb095248f307cc7bac7b0268ae4d5d3440f300/src/Standards/Squiz/Sniffs/NamingConventions/ValidVariableNameSniff.php#L129